gui-tk/minsky.tcl

```diff
@@ -1464,7 +1477,7 @@ proc aboutMinsky {} {
    be purchased to use Ravel. See https://ravelation.hpcoders.com.au
 
 Thanks to following Minsky Unicorn sponsors:
-     Edward McDaniel
```

Comment:
Please merge with master. This undoes a change I made last night.

Reply:
I don't understand why changes from different branches get committed along with the changes in this branch. I thought git was smart enough to separate these things, otherwise I don't understand why branches even exist? Before committing this change I pulled the most-up-to-date master branch into this bugfix, so i also do not know why it undoes changes you made? What do I do, revert this change and pull the latest version into my local bugfix branch and push again before making any code changes?